src/main/webapp/map.js

```diff
@@ -153,12 +156,45 @@ function addWaterSystem(areaType, zipCode){
             console.log(system);
             console.log(system.contaminants);
             waterPollutionHTML += 
-                "<h2>"+system.name+"</h2>"+
-                "<h4>Serves "+system.populationServed+"</h4>";
+                "<div id='"+system.pwsid+"' class='water_pollution'>"+
```

Comment:
nit: Again, no action required, but something for you to learn. Usually you don't hard code HTML into JS , but rather have template files and use a templating engine to generate the final HTML to inject into a page. See https://github.com/janl/mustache.js for an example.

Reply:
That makes a lot of sense, it felt strange to write it like that